department-of-veterans-affairs/va.gov-cms
# VACMS-3840: Appt-Intro-Text

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3840

## Testing done


## Screenshots


## QA steps

- Login to preview environment
- Navigate here: `/node/add/health_care_local_health_service`
- Fill out required fields
- Leave Appointment Intro Text selection as : `Use default appointments intro text`
- [x] Confirm you can see the default intro text below the selections as   _Contact us to schedule, reschedule, or cancel your
  appointment. If a referral is required, you’ll need to
  contact your primary care provider first._
- Save

- Confirm the following on the preview:
- [x] APPOINTMENT INTRO TEXT
Use default appointments intro text
- [x] APPOINTMENT LEAD-IN DEFAULT
  Contact us to schedule, reschedule, or cancel your
  appointment. If a referral is required, you’ll need to
  contact your primary care provider first.

- Navigate here: `/node/add/health_care_local_health_service`
- Fill out required fields
- Select Appointment Intro Text selection as : `Customize appointments intro text`
- [x] Confirm you can see an empty input field
- Save without filling the field out.
- Confirm the following error message on save: `1 error has been found: Appointment lead-in text`
- Fill out of the empty field: `test`
- Save

- Confirm the following on the preview:
- [x] APPOINTMENT INTRO TEXT
Customize appointments intro text
- [x] APPOINTMENT LEAD-IN TEXT
test

- Navigate here: `/node/add/health_care_local_health_service`
- Fill out required fields
- Change Appointment Intro Text selection as : `No appointments intro text`
- [x] Confirm that the default appt intro text hides and that you cannot see the following text: _Contact us to schedule, reschedule, or cancel your
  appointment. If a referral is required, you’ll need to
  contact your primary care provider first._
- Save

- Confirm the following on the preview:
- [x] APPOINTMENT INTRO TEXT
No appointments intro text

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
